azure-pipelines.yml

```diff
@@ -48,6 +48,8 @@ variables:
     value: RelWithDebInfo
   - name: LIBWEBRTC_VERSION
     value: 112.0
```

Comment:
I found there are 3 '112' for webRTC version, I suggest we have a constant to define the webRTC version. So we only need to modify once.

Reply:
The major one seems to be LIBWEBRTC_VERSION. If next both (LIBWEBRTC_RELEASE_TAG and OBS_VERSION) have defined naming convention then it is reasonable. Otherwise, we will need to update them anyway, in spite of their lilbwebrtc version included. I do not know that, so if you are certain of that then I can implement it.